### PR TITLE
Added 3 apps

### PR DESCRIPTION
**Pull request by Snowglob**
#### Added Lemuroid to "Games > Retro", Added Trail Sense to "Maps and Navigation", Added PDF Converter to " Utilities"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -292,6 +292,7 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 * [**OsmAnd~**](http://osmand.net/) <sup>**[[F-Droid](https://f-droid.org/app/net.osmand.plus)]**</sup>
     * [Contour Lines and Hillshade](http://osmand.net/features?id=contour-lines-plugin) <sup>**[[F-Droid](https://f-droid.org/app/net.osmand.srtmPlugin.paid)]**</sup>
     * [Parking Spot](http://osmand.net/features?id=parking-plugin) <sup>**[[F-Droid](https://f-droid.org/app/net.osmand.parkingPlugin)]**</sup>
+* [**Trail Sense**](https://kylecorry.com/app/maker/subnautica/2020/05/14/trail-sense.html) <sup>**[[F-Droid](https://f-droid.org/en/packages/com.kylecorry.trail_sense/)]**</sup>
```
Comment (offa) [inline README.md:295]: Trail Sense is included under *Misc* already. But I do not have any preference where to put, so if it suits better here, just remove it in the other section.

Comment (Snowglob) [reply README.md:295]: OK, will do 👍

Comment (Snowglob) [reply README.md]: Done. Check the newest commit

**Comment by offa** [comment]
Perfect, thank you! :smile:

**Review by offa — APPROVED** [review]

**Comment by offa** [comment]
Sorry, CI builds didn't trigger automatically. I'll fix the reported issues.

**Comment by Snowglob** [comment]
Its fine. This is my first time contributing so I'm pretty excited for this!

**Comment by offa** [comment]
Nice, then welcome to Open Source development! :partying_face: :tada:

**Comment by Snowglob** [comment]
Thank you!

**Pull request merged**